stock_average_daily_sale/views/stock_warehouse.xml

```diff
@@ -10,7 +10,11 @@
         <field name="arch" type="xml">
             <xpath expr="//group[1]" position="after">
                 <group string="Average Sales" name="average_sale_reporting">
-                    <field name="average_daily_sale_root_location_id" />
+                    <field name="lot_stock_id" invisible="1" />
```

Comment:
I needed to add this field to the view because of the error `Field 'lot_stock_id' used in attrs ({'required': [('lot_stock_id', '!=', False)]}) is restricted to the group(s) stock.group_adv_location,stock.group_stock_multi_warehouses,base.group_no_one.' I've got this error after adding the required attribute to the field below. Adding the abovementioned groups to the `average_daily_sale_root_location_id' field or to it's parent view group doesn't solve the issue while adding invisible `lot_stock_id` field does.